Formula/notcurses.rb

```diff
@@ -0,0 +1,32 @@
+class Notcurses < Formula
+  desc "Blingful character graphics/TUI library"
+  homepage "https://nick-black.com/dankwiki/index.php/Notcurses"
+  url "https://github.com/dankamongmen/notcurses/archive/refs/tags/v2.3.12.tar.gz"
+  sha256 "ce042908fac11f7df1f9eaa610e46e9c615f53ab036b7c27ae2396292512407b"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "doctest" => :build
+  depends_on "pandoc" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ffmpeg"
+  depends_on "libunistring"
+  depends_on "ncurses"
+  depends_on "readline"
+  uses_from_macos "zlib"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DCMAKE_INSTALL_RPATH=#{rpath}"
```
I think this should fix your `image not found` error.